src/index.html

```diff
@@ -1,15 +1,195 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    onsubmit="onSubmit()">
+
+      <!------------------ Personal information block ------------------>
+      <fieldset class="formTable">
+        <legend>Personal information</legend>
+
+        <!-- Surname field -->
+        <div class="formfield">
+          <label for="surname">Surname:</label>
+          <input type="text"
+          id="surname"
+          name="surname"
+          autocomplete="off">
```

Comment:
І тут так само і з всіма інпутами так само.
```suggestion
          <input 
            type="text"
            id="surname"
            name="surname"
            autocomplete="off"
          >
```

Reply:
Done